models/cam-fv/readme.rst

```diff
@@ -6,7 +10,7 @@ Overview
 
 The larger context of the Community Earth System Model and DART interactions
 is described in the `CESM readme <../CESM/readme.html>`_
-This document focuses on the several `atmospheric models <http://www2.cesm.ucar.edu/models>`__
+This document focuses on the several `atmospheric models <http://www2.cesm.ucar.edu/models>`_
```

Comment:
Are you updating all the links in this document?

If so, <../CESM/readme.html>`_ should be changed to not be html so it builds correctly with latex, etc. builds of the documentation.

Reply:
My goal was not to update all the links, but it seemed like I should fix link-related things that I noticed. My intent was the same as the Publications change (previous comment), but you've brought up a new issue - latex - so I'm wondering whether the html form is fine in an anonymous link (__) but not in a named link (_).

Reply:
I'd probably enjoy making it super tidy and maintainable, but I don't want to add that mission creep to this issue.  I've opted for the :doc: form (in 3 places).  Thanks for the tips!